src/components/Forms/ScheduleForm/ScheduleForm.js

```diff
@@ -143,13 +113,7 @@ const ScheduleForm = (props) => {
           className="form__input form__textarea"
           {...message.bind}></textarea>
       </div>
-      <Button
-        type="submit"
-        className={`form__button ${buttonClass}`}
-        isDisabled={isError || isSuccess || isLoading ? true : false}
-        text={buttonText}
-        icon={buttonIcon}
-      />
+      <Button type="submit" {...formStatus.bindButton} />
```

Comment:
Instead of spreading the props here, I think it would be better to set them individually. It's makes it easier to reason about. Also, we have this rule on in eslint: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md